allauth_2fa/utils.py

```diff
@@ -0,0 +1,42 @@
+from base64 import b32encode
+
+from django.contrib.sites.shortcuts import get_current_site
+from django.utils.six import BytesIO
+
+import qrcode
+from qrcode.image.svg import SvgPathImage
```

Comment:
Does this require any additional packages to be installed? (I assume not, but just checking!)

Reply:
Nope!

It's what the original code uses anyway.

https://github.com/percipient/django-allauth-2fa/blob/a63008ff79049942fcafcc72a856ee13722ea35e/allauth_2fa/views.py#L24